src/auth/permissions/__tests__/index.test.integration.ts

```diff
@@ -55,13 +58,21 @@ describe('Permisions Module', () => {
   describe('::grantExisting', () => {
     test('SUCCESS - Grant a role a permission entry that already exists', async () => {
       try {
+        const permissionId = await knex('permission').select('permission_id').where({
+          permission_entity: Resource.CONSTANTS,
+          access_type: Access.WRITE,
+          permission_level: PermissionLevel.ALL,
+        }).then((x) => x[0].permission_id);
```

Comment:
If you only need the first, you can add `.limit(1)` onto the query. Then you can destructure the id directly.

Reply:
Knex returns an array even if you use `.limit()`